frappe/email/inbox.py

```diff
@@ -0,0 +1,79 @@
+import frappe
+import json
+
+def get_email_accounts(user=None):
+	if not user:
+		user = frappe.session.user
+
+	email_accounts = []
+
+	accounts = frappe.get_all("User Email", filters={ "parent": user },
+		fields=["email_account", "email_id"],
+		distinct=True, order_by="idx")
+
+	if not accounts:
+		return {
+			"email_accounts": [],
+			"all_accounts": ""
+		}
+
+	email_accounts.append({
```

Comment:
@RobertSchouten, Already added the condition in communication.js, it will ignore the Sent and All Accounts in Communication Composer